src/task/pool.rs

```diff
@@ -111,28 +111,26 @@ impl Pool {
 
 #[inline]
 pub(crate) fn get() -> &'static Pool {
-    lazy_static! {
-        static ref POOL: Pool = {
-            let num_threads = num_cpus::get().max(1);
-            let mut stealers = Vec::new();
+    static POOL: Lazy<Pool> = Lazy::new(|| {
```

Comment:
As this doesn't really make heavy use of `Deref` for Lazy, a more minimal and direct approach would be to use OnceCell.

```rust
static POOL: OnceCell<Pool> = OnceCell::new();
POOL.get_or_init(|| ...)
```

But, as we are using `Lazy` elsewhere, I guess it's better to use it here as well, just for consistency

Reply:
Yeah, I make use of `Lazy` mostly to streamline with on-going rust RFC. I think this will be easier for later maintenance.

Reply:
The RFC proposes both `OnceCell` and `Lazy`, as `OnceCell` is strictly more powerful.